project/Dependencies.scala

```diff
@@ -37,8 +37,8 @@ object Dependencies extends AutoPlugin {
 
       val version = "1.4.1"
       val api = "io.opentelemetry" % "opentelemetry-api" % version
-      val sdk = "io.opentelemetry" % "opentelemetry-sdk" % OpenTelemetry.version % Test
-      val sdkTesting = "io.opentelemetry" % "opentelemetry-sdk-testing" % OpenTelemetry.version % Test
+      val sdk = "io.opentelemetry" % "opentelemetry-sdk" % OpenTelemetry.version
+      val sdkTesting = "io.opentelemetry" % "opentelemetry-sdk-testing" % OpenTelemetry.version
```

Comment:
These might be merge conflicts of some kind? the opentelemetry-sdk stuff should be just test dependencies though - libraries should only depend on the API.

Reply:
ActorWithTracing depends on `io.opentelemetry.api.trace.{ Span, Tracer }`

Reply:
I think those both come from the `opentelemetry-api` dependency

Reply:
done